contrib/kustomize/device-injector/daemonset.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: nri-plugin-device-injector
+spec:
+  template:
+    spec:
```

Comment:
for some of the plugins, `priorityClassName: system-node-critical` might make sense?